projects/spring-framework/JdbcCoreMapperFuzzer.java

```diff
@@ -0,0 +1,111 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
```

Comment:
Please add Copyright header.

Reply:
Added in https://github.com/google/oss-fuzz/pull/8359/commits/3fee0cfc8fd8ee7f05701fd86ae1a717d7324c03